plugins/hypervisors/kvm/src/main/java/org/apache/cloudstack/utils/qemu/QemuImageOptions.java

```diff
@@ -56,13 +62,15 @@ public QemuImageOptions(QemuImg.PhysicalDiskFormat format, String filePath, Stri
             }
         }
         if (format != null) {
-            params.put("driver", format.toString());
+            params.put(DRIVER, format.toString());
+            this.format = format;
         }
```

Comment:
we can change this to 
```suggestion
        setFormat(format);
```

Reply:
@DaanHoogland 
good point. thanks
applied